### PR TITLE
Feature/carbon 153

### PR DESCRIPTION
**Pull request by dhanushkaxep**

### CODE REVIEW

File: backend/services/src/national-api/national.api.module.ts [hunk 1]
```diff
@@ -35,7 +36,7 @@ import { UtilModule } from '../shared/util/util.module';
     CompanyModule,
     UtilModule
   ],
-  controllers: [ NationalAPIController, UserController, AuthController, CompanyController, ProgrammeController ],
+  controllers: [ NationalAPIController, UserController, AuthController, CompanyController, ProgrammeController, settingsController ],
```
Comment (palindaa) [inline backend/services/src/national-api/national.api.module.ts:39]: s should be capital

File: backend/services/src/setup/handler.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ import { LocationInterface } from "../shared/location/location.interface";
 import { ProgrammeModule } from "../shared/programme/programme.module";
 import { ProgrammeService } from "../shared/programme/programme.service";
 import { ConfigService } from "@nestjs/config";
+import {ConfigurationSettingsService } from '../shared/util/configurationSettings.service'
+import { ConfigurationSettingsType } from "src/shared/enum/configuration.settings.type.enum";
```
Comment (palindaa) [inline backend/services/src/setup/handler.ts:26]: src/ path should be relative

File: backend/services/src/setup/handler.ts [hunk 3]
```diff
@@ -213,6 +216,8 @@ export const handler: Handler = async (event) => {
     }
   });
 
+  configurationSettingsService.saveSetting(ConfigurationSettingsType.isTransferFrozen, false);
```
Comment (palindaa) [inline backend/services/src/setup/handler.ts:219]: What if we does not save this. What is the default behaviour? If this does not save, behaviour should be false

File: backend/services/src/shared/util/configurationSettings.service.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+@Injectable()
+export class ConfigurationSettingsService {
+  constructor(
+    @InjectRepository(ConfigurationSettings)
+    private configSettingsRepo: Repository<ConfigurationSettings>,
+    private logger: Logger,
+    private helperService: HelperService
+  ) {}
+
+  async getSetting(type: number) {
```
Comment (palindaa) [inline backend/services/src/shared/util/configurationSettings.service.ts:18]: If you add optional parameter to parse default value and if the value is null return the default value will simplify default behaviour handle

File: backend/services/src/national-api/national.api.module.ts (hunk 1 above)
Comment (dhanushkaxep) [reply backend/services/src/national-api/national.api.module.ts]: fixed

File: backend/services/src/setup/handler.ts (hunk 2 above)
Comment (dhanushkaxep) [reply backend/services/src/setup/handler.ts]: removed

File: backend/services/src/setup/handler.ts (hunk 3 above)
Comment (dhanushkaxep) [reply backend/services/src/setup/handler.ts]: removed from setup handler

File: backend/services/src/shared/util/configurationSettings.service.ts (hunk 4 above)
Comment (dhanushkaxep) [reply backend/services/src/shared/util/configurationSettings.service.ts:18]: fixed

**Review by palindaa — APPROVED** [review]

**Pull request merged**